src/Qowaiv/Clock.cs

```diff
@@ -36,14 +38,14 @@ public static class Clock
     [Pure]
     public static DateTime UtcNow()
     {
-        var utcNow = (threadUtcNow ?? globalUtcNow).Invoke();
+        var utcNow = (threadUtcNow.Value ?? globalUtcNow).Invoke();
         return utcNow.Kind == DateTimeKind.Utc
             ? utcNow
             : new DateTime(utcNow.Ticks, DateTimeKind.Utc);
     }
 
     /// <summary>Gets the time zone of the <see cref="Clock"/>.</summary>
-    public static TimeZoneInfo TimeZone => threadTimeZone ?? globalTimeZone;
+    public static TimeZoneInfo TimeZone => threadTimeZone.Value ?? globalTimeZone;
```

Comment:
Can threadTimeZone be null?

Reply:
Yep. It can. I previously was a `Func` that could be null. Now it is a `LocalAsync<T>` where the `Value` can be null.